CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v2.0.0 - [xxxx-xx-xx]
```

Comment:
To go from v1 to v2 in semantic versioning would be kind of bad, that signifies a major breaking change.

Typically next up would be v1.0.1 (we fixed a bug) or v1.1.0 (we added some functionality)

```suggestion
## v0.1.0 - [xxxx-xx-xx]
```